### DISCUSSION TITLE
With the release of v4, will there be an ESLint plugin akin to the current `prettier-plugin-tailwind`?

### DISCUSSION THREAD

**Discussion by stefanobartoletti** [tailwindlabs/tailwindcss#13299] [Ideas]
Pretty much like the title says, is an ESLint version of the Prettier plugin planned in the near (or not so near) future?

ESLint is a popular tool and many people prefer to use it alone, instead of coupling it with the strongly opinionated Prettier.

At the moment there is a third-party plugin, https://github.com/francoismassart/eslint-plugin-tailwindcss, but it has not received many updates lately, as the author (and from what I know the only maintainer) doesn't seem to be very active (not complaining, I'm grateful for the great job he did, and open-source is "as-is").

There are a couple of open discussions about both the future of that plugin itself (https://github.com/francoismassart/eslint-plugin-tailwindcss/issues/295), and its possible support of Tailwind 4 (https://github.com/francoismassart/eslint-plugin-tailwindcss/issues/325), but the author stated that he is able to work on it only in its spare time. 

This was to give some context, are there some plans for an official plugin?

**Comment by techotaku1** [comment]
si eso quiero saber porque ya hize la migracion de tailwind 3 ala 4 pero nose si slint seguira soportando en esta version de tailwind 4.0?
